src/main/java/Main.java

```diff
@@ -1,5 +1,113 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
 public class Main {
+
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_BLUE = "\u001B[34m";
+    public static final String ANSI_MAGENTA = "\u001b[35m";
+
+    public static JSONObject startRoom;
+    public static JSONObject endRoom;
+
+    public static HashMap<String, Room> roomMap = new HashMap<>();
+    public static HashMap<String, Item> itemMap = new HashMap<>();
+
+    public static String playerName;
+
     public static void main (String[] args){
```

Comment:
The  main method has a lot of content. A lot of the functionality from main could've been put in separate methods which would make your code more readable and easier to work with and debug.